index.html

```diff
@@ -152,7 +152,47 @@ <h2>Multi-Post Stories</h2>
           <li>css</li>
           <li>javaScript</li>
         </ul>
-        <button type="button">See Project</button>
+        <button type="button" onclick="getData(4)" id="popupButton">See Project</button>
+      </div>
+    </div>
+    <div class="popup-container" id="popupContainer">
+      <div class="popup-wrapper">
+        <div class="popup-sub-wrapper">
+          <div class="popup-header">
+            <div>
+              <h2 id="popupTitle">Title</h2>
+              <div class="work-desc-wrapper-div">
+                <span class="work-desc-wrapper-position work-desc-wrapper-item" id="popupPlace">Place</span>
+                <i></i>
+                <span class="work-desc-wrapper-item" id="popupPosition">Position</span>
+                <i></i>
+                <span class="work-desc-wrapper-item" id="popupPeriod">Period</span>
+              </div>
+            </div>
+            <i class="fa-solid fa-xmark" id="popupClose"></i>
+          </div>
+          <div class="popup-img"><img src="./images/art.png" alt="work pictictre" class="work-place-img" id="popupImage" /></div>
+          <div class="popup-skills-cont">
+            <p id="popupDescription"></p>
+            <div class="popup-skills-badge">
+              <ul id="popupSkills"></ul>
+              <div class="popup-link-main-cont">
+                <a href="#" target="_blank" class="popup-link-cont" id="liveDemo">
+                  <span class="popup-link">
+                See live
+                <i class="fa-solid fa-up-right-from-square"></i>
+              </span>
+                </a>
+                <a href="#" target="_blank" class="popup-link-cont" id="sourceCode">
+                  <span class="popup-link">
+                <span>See Source</span>
+                  <i class="fa-brands fa-github"></i>
+                  </span>
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] Kindly add the entire Projects section dynamically. Kindly make changes in existing code by removing already created HTML code for the projects list, Instead, it should be added via JavaScript dynamically.